src/main/java/training/endava/playground/generics/types/Article.java

```diff
@@ -0,0 +1,49 @@
+package training.endava.playground.generics.types;
+
+import java.util.ArrayList;
+
+public class Article {
+
+    private String title;
+    private String author;
+    private ArrayList<String> tag;
```

Comment:
If there is a list of tags please name the field tags just to be clear about this